DCode.Web/Scripts/App_JS/Controllers/Requestor.js

```diff
@@ -82,6 +82,7 @@
                 $scope.dashboard.showApproval = false;
                 $scope.dashboard.showTaskStatus = false;
                 $scope.dashboard.showHistory = true;
+                $("#projectNam").attr("placeholder", "Search by Project Name/Firm Initiative");

```

Comment:
This change would not be required. Please refer to the comment on `Dashboard.cshtml`

Reply:
As discussed with @priyankalekkala , this change along with the one in Dashboard.cshtml is necessary. Hence, the change isn't required.